models/MITgcm_ocean/create_ocean_obs.f90

```diff
@@ -1,8 +1,6 @@
 ! DART software - Copyright UCAR. This open source software is provided
```

Comment:
agree that this file and the dart_MITocean_mod belong in an observations/obs_converters directory.  also, i don't see anything MITocean specific in that module.  rename to something like read_ocean_obs_mod.f90 or something more specific to the source of the file format.

Reply:
Is create_ocean_obs actually used?  The first line of the create_ocean_obs documentation is 

```
create_ocean_obs is responsible for converting an interim ASCII file of ocean observations into a DART observation
sequence file. The interim ASCII file is a simple 'whitespace separated' table where each row is an observation and each
column is specific information about the observation.
```

but the ocean_color converter and the WOD converter write obs_sequence files.   Can create_ocean_obs be removed?